restfiles/trigger-gitlab.restfile

```diff
@@ -0,0 +1,18 @@
+variables:
+    base_url: https://gitlab.com/api/v4/projects/19564054
+    pipeline_token: <...>
+    version_id: <...>
+
+requests:
+
+    trigger:
+        url: ${base_url}/trigger/pipeline
+        method: POST
+        body:
+            form:
+                token: ${pipeline_token}
+                ref: main
+                variables[foo]: bar
+        validation:
+            status: 201
+        log: json
```

Comment:
This is a test file used to implement `Gitlab.Builder.postTrigger` in `Gitlab.swift`. Just keeping it around for future testing/exploration.